src/components/facets/Facet.tsx

```diff
@@ -1,17 +1,19 @@
 import * as React from 'react';
 import * as _ from 'underscore';
-import { IReduxStatePossibleProps } from '../../utils/ReduxUtils';
-import { Svg } from '../svg/Svg';
-import { Tooltip } from '../tooltip/Tooltip';
-import { FacetMoreRows } from './FacetMoreRows';
-import { FacetMoreRowsConnected } from './FacetMoreRowsConnected';
-import { FacetMoreToggle } from './FacetMoreToggle';
-import { FacetMoreToggleConnected } from './FacetMoreToggleConnected';
-import { FacetRow } from './FacetRow';
+import {IReduxStatePossibleProps} from '../../utils/ReduxUtils';
+import {Svg} from '../svg/Svg';
+import {Tooltip} from '../tooltip/Tooltip';
+import {FacetMoreRows} from './FacetMoreRows';
+import {FacetMoreRowsConnected} from './FacetMoreRowsConnected';
+import {FacetMoreToggle} from './FacetMoreToggle';
+import {FacetMoreToggleConnected} from './FacetMoreToggleConnected';
+import {FacetRow} from './FacetRow';
 
 export interface IFacet {
   name: string;
   formattedName: string;
+  tooltipName?: string;
```

Comment:
IMHO `tooltipName` should be renamed to `tooltipLabel` or something similar, because _"name"_ could confuse someone into thinking this refers to the `name` attribute of the html tag used for the tooltip.